### ISSUE TITLE
Javascript/TypeScript language service is not starting with remote ssh

### ISSUE THREAD

**Issue by nanot1m** [microsoft/vscode-remote-release#380]
After upgrading to VSCode 1.35.0-insider Javascript/TypeScript language stopped working.
Removing/reinstalling VSCode and extensions does not help

- VSCode Version: **1.35.0-insider**
- Local OS Version: **macOS Mojave 10.14.4**
- Remote OS Version: **0.37**
- Remote Extension/Connection Type: **SSH**

- Does this issue occur when you try this locally?: **No**
- Does this issue occur when you try this locally and all extensions are disabled?: **Yes**

**Comment by nanot1m** [comment]
I've found out there was no access to `/var/tmp/vscode-typescript/` folder on remote resource. After granting permissions problem disappeared

**Comment by houkanshan** [comment]
@nanot1m Thanks, how did you find that? Is there any log for it? 

I can't granting the permission of /var/tmp so I tried to change my `$TMPDIR` on the remote server, and VSCode SSH used it then created a `vscode-remote-install.xxx.sh` file in the new tmp dir. But the Typescript language service was still not working. No other new file was created in the new tmp dir.

The only log I got from `OUTPUT - Typescript` is
```
[Info  - 11:32:41] Using tsserver from: /home/xxx/node_modules/typescript/lib
```
